probe_scraper/parsers/scalars.py

```diff
@@ -17,7 +17,6 @@ def extract_scalar_data(s):
     return {
         "description": s.description,
         "expiry_version": get_major_version(s.expires),
-        "cpp_guard": s.cpp_guard,
```

Comment:
vendored parser removed `cpp_guard` from the list optional fields for scalars

Reply:
We sure did. It kinda sucked. I sure hope this doesn't trip up any historical parsing (if it did, the tests would've caught it, right?).

Reply:
unit tests failed when the `cpp_guard` field was present in a scalar.

note to future relud: do a test run of probe scraper to see if this will be an issue before merging.

Reply:
or I could patch `cpp_guard` back in

Reply:
It does seem like that'd be the smaller diff, true

Reply:
I'll do that then. I don't want to break things while I'm out, though, so converting this to a draft until after US thanksgiving.

Reply:
i misunderstood what was going on. unit tests pass fine when `cpp_guard` is present in the scalar definition.

they only failed because it was missing from the parser output, which is expected.